app/controllers/static_controller.rb

```diff
@@ -0,0 +1,5 @@
+class StaticController < ApplicationController
+  def about
+    @pokemons = ['stache.png']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.